Modules/_sqlite/connection.c

```diff
@@ -1339,89 +1347,107 @@ PyObject* pysqlite_connection_call(pysqlite_Connection* self, PyObject* args, Py
     return NULL;
 }
 
-PyObject* pysqlite_connection_execute(pysqlite_Connection* self, PyObject* args)
+/*[clinic input]
+_sqlite3.Connection.execute as pysqlite_connection_execute
+
+    sql: unicode
+    /
+    parameters: object = NULL
```

Comment:
Looks wrong, execute does not accept keyword arguments

Reply:
Correct. I'll fix.